ledger/participant-state/kvutils/tools/src/main/scala/com/daml/ledger/participant/state/kvutils/tools/IntegrityCheck.scala

```diff
@@ -115,7 +115,10 @@ object IntegrityCheck extends App {
         assert(updates.head.isInstanceOf[Update.CommandRejected])
       case Proto.DamlLogEntry.PayloadCase.PACKAGE_UPLOAD_ENTRY =>
         // NOTE(JM): Possibly empty.
-        assert(updates.forall(_.isInstanceOf[Update.PublicPackageUploaded]))
+        assert(
+          updates.forall(update =>
+            update.isInstanceOf[Update.PublicPackageUploaded] || update
+              .isInstanceOf[Update.PackageUploadEntryAccepted]))
```

Comment:
Does this also need the clause for `Update.PackageUPloadEntryRejected`?

Reply:
yes good point -- that should be there too but probably was not exercised / evident because the semantic test that is run to gather this ledger dump does not include any PackageUploadEntryRejected items

Reply:
added Rejected and typ is now typed